### DISCUSSION TITLE
Missing network field in custom chains

### DISCUSSION THREAD

**Discussion by railBitsAndHex** [rainbow-me/rainbowkit#447] [Q&A]
When using **custom chains** in **Typescript**, following the content as per mentioned in the docs

![image](https://user-images.githubusercontent.com/104562337/171042025-76d5c0e7-b7a6-4cb5-8e52-9225a5475d99.png)

I noticed that I recieved the following error **(ts2332)**

````Property 'network' is missing in type '{ id: number; name: string; nativeCurrency: { decimals: number; name: string; symbol: string; }; rpcUrls: { default: string; }; blockExplorers: { default: { name: string; url: string; }; snowtrace: { name: string; url: string; }; }; testnet: false; }' but required in type 'Chain'.````

It seems that the `network` field is missing and not mentioned in the sample code from the docs, but necessary to be added.

May I know if the docs can be updated to prevent this issue from occurring?

**Comment by markdalgleish** [comment]
Thanks for spotting this! I also noticed that there was a type error with `blockExplorers` too. I just opened a PR fixing this: https://github.com/rainbow-me/rainbowkit/pull/449

This example was borrowed from the wagmi docs, so I've sent them a PR too: https://github.com/tmm/wagmi/pull/532

**Reply by railBitsAndHex** [reply]
Thank you 👍
